internal/overlord/checkstate/checkers.go

```diff
@@ -75,7 +75,7 @@ func (c *httpChecker) check(ctx context.Context) error {
 			details = strings.Join(lines, "\n")
 		}
 		return &detailsError{
-			error:   fmt.Errorf("received non-20x status code %d", response.StatusCode),
+			error:   fmt.Errorf("non-20x status code %d", response.StatusCode),
```

Comment:
I shortened this check error message slightly as "received" seemed redundant.

Reply:
usually we call them 2xx 3xx 4xx etc when referring to http status codes

Reply:
Good point. Fixed.